### PR TITLE
Allow base64-encoded image 'paths'

### PR DESCRIPTION
**Pull request by clintandrewhall**
This change allows base-64 encoded 'paths' to be placed in the JSON layer file, allowing single-page rendering of Framer prototypes.

### CODE REVIEW

File: framer/Importer.coffee [hunk 1]
```diff
@@ -70,13 +70,16 @@ class exports.Importer
 		# Most layers will have an image, add that here
 		if info.image
 			layerInfo.frame = info.image.frame
-			layerInfo.image = Utils.pathJoin @path, info.image.path
-			
+			if info.image.path.indexOf('data:image') != -1
```
Comment (koenbok) [inline framer/Importer.coffee]: I think it's a bit nicer to use https://lodash.com/docs#startsWith here.

Comment (koenbok) [inline framer/Importer.coffee]: I think it's a bit more readable/correct to use https://lodash.com/docs#startsWith here.

File: framer/Layer.coffee [hunk 2]
```diff
@@ -513,7 +513,7 @@ class exports.Layer extends BaseClass
 
 			# If the file is local, we want to avoid caching
 			# if Utils.isLocal() and not (_.startsWith(imageUrl, "http://") or _.startsWith(imageUrl, "https://"))
-			if Utils.isLocal() and not imageUrl.match(/^https?:\/\//) and @_cacheImage is false
```
Comment (koenbok) [inline framer/Layer.coffee]: Here too.

**Comment by koenbok** [comment]
So I guess the idea is to have one self contained json file with everything? I'm not completely sure what it's useful for, but I like experiments :-)

**Comment by clintandrewhall** [comment]
Hi Koen!  Just a head's up: I sent you a message on Facebook, not sure if it went to your "other" inbox. :-)  Figured it would be faster just to explain over IM...

**Comment by koenbok** [comment]
Can I still help with this?

**Comment by jordandobson** [comment]
It would be great to get this in... I have a hacky way of doing this at the moment... but I'm not sure I need a JSON file either... does it really matter where they are stored?

**Comment by koenbok** [comment]
Closing due to inactivity. I'm happy to help and land it at some point.

**Pull request closed without merging**